apps/console/src/features/users/components/wizard/add-user-wizard.tsx

```diff
@@ -744,12 +972,392 @@ export const AddUserWizard: FunctionComponent<AddUserWizardPropsInterface> = (
         }
     ];
 
+        /**
+     * Resolves the step content.
+     *
+     * @returns Step content.
+     */
+    const resolveWizardTitle = (): string => {
+        let wizardTitle: string = "";
+
+        if (defaultUserTypeSelection === UserAccountTypes.USER) {
+            userTypeSelection === (UserAccountTypesMain.INTERNAL) 
+            ? wizardTitle += t("extensions:manage.users.wizard.addUser.title") 
+            : wizardTitle += t("console:manage.features.parentOrgInvitations.addUserWizard.heading") 
+        } 
+        
+        if (defaultUserTypeSelection === UserAccountTypes.ADMINISTRATOR) {
+            adminTypeSelection === AdminAccountTypes.INTERNAL
+                ? wizardTitle += t("extensions:manage.users.wizard.addAdmin.internal.title")
+                : wizardTitle += t("extensions:manage.users.wizard.addAdmin.external.title");
+        }
+
+        if (wizardState && wizardState[ WizardStepsFormTypes.BASIC_DETAILS ]?.firstName) {
+            wizardTitle += " - " + wizardState[ WizardStepsFormTypes.BASIC_DETAILS ]?.firstName;
+        } else {
+            wizardTitle += "";
+        }
+
+        return wizardTitle;
+    };
+
+    const resolveWizardSubHeading = (): string => {
+        let wizardSubHeading: string = "";
+        if (defaultUserTypeSelection === UserAccountTypes.USER) {
+            (userTypeSelection === UserAccountTypesMain.INTERNAL)
+            ? wizardSubHeading += t("extensions:manage.users.wizard.addUser.subtitle")
+            : wizardSubHeading += t("console:manage.features.parentOrgInvitations.addUserWizard.description")
+        }
+        return wizardSubHeading
```

Comment:
Add a newline before `return`.
```suggestion

        return wizardSubHeading
```